README.md

```diff
@@ -1 +1,10 @@
-# ss-deploy
\ No newline at end of file
+# ss-deploy
+
+## Setup
+
+1. Copy the `.env.example` file to `.env`:
```

Comment:
the file name is actually `sample.env` and for simplicity we can rename it by dropping "sample" in the name resulting in `.env`